src/sentry/static/sentry/app/views/issueList/header.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+import QueryCount from 'app/components/queryCount';
+import * as Layout from 'app/components/layouts/thirds';
+
+type Props = {
+  query: string;
+  queryCount: number;
+  queryMaxCount: number;
+  onTabChange: (query: string) => void;
+};
+
+const queries = [
+  ['is:inbox', t('Inbox')],
+  ['is:unresolved', t('Backlog')],
+  ['is:ignored', t('Ignored')],
+  ['is:resolved', t('Resolved')],
+];
+
+function IssueListHeader({query, queryCount, queryMaxCount, onTabChange}: Props) {
+  const count = <StyledQueryCount count={queryCount} max={queryMaxCount} />;
+
+  return (
+    <React.Fragment>
+      <BorderlessHeader>
+        <StyledHeaderContent>
+          <StyledLayoutTitle>{t('Issues')}</StyledLayoutTitle>
+        </StyledHeaderContent>
+      </BorderlessHeader>
+      <TabLayoutHeader>
+        <Layout.HeaderNavTabs underlined>
+          {queries.map(([tabQuery, queryName]) => (
+            <li key={tabQuery} className={query === tabQuery ? 'active' : ''}>
+              <a onClick={() => onTabChange(tabQuery)}>
+                {queryName} {query === tabQuery && count}
```

Comment:
Are we sure we want to do this? In designs we have Inbox having the count as well as the active tab.

Reply:
we'd have to make another query to always show the count on the inbox, can take a look at that in another PR

Reply:
ok sounds good